### DISCUSSION TITLE
Would it be possible to add gender detection?

### DISCUSSION THREAD

**Discussion by RuairiRobinson** [Bing-su/adetailer#387] [Ideas]
This feature has been added to ReActor and works really well and would alleviate one of the biggest frustrations with Adetailer -- there's no good way to allow SD to produce bigger composition variations but to apply adetailer to one sex/gender only. 

Thanks!
R.

**Comment by SarahPeterson2854** [comment]
bump

**Reply by Gh20221205** [reply]
Bump. This is a much needed feature for image with both genders and I want to repaint faces correctly.

**Reply by fg-uulm** [reply]
I created a prototype which can do this (and many other conditional inpainting scenarios), please see and comment on issue #763

**Comment by SarahPeterson2854** [comment]
> This feature has been added to ReActor and works really well and would alleviate one of the biggest frustrations with Adetailer -- there's no good way to allow SD to produce bigger composition variations but to apply adetailer to one sex/gender only.
> 
> Thanks! R.

fyi all this has been done for a while now thanks to a commissioner: 

Works well, male face, female face. 

https://civitai.com/models/293448/anzhcs-face-segmentation-prototype-or-yolov8-or-adetailer-model
